src/Commands/SyncUnionCloudDataUsers.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace BristolSU\UnionCloud\Commands;
+
+use BristolSU\UnionCloud\Jobs\getUsersData;
+use BristolSU\UnionCloud\Jobs\notifyAdmin;
+use BristolSU\UnionCloud\Jobs\processUserData;
+use GuzzleHttp\Exception\ClientException;
+use Illuminate\Console\Command;
+use BristolSU\UnionCloud\UnionCloud\UnionCloud;
+use Illuminate\Support\Facades\Log;
+
+
+class SyncUnionCloudDataUsers extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'unioncloud:users:sync:all';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sync users from UnionCloud';
+
+    // Not currently working as Page Limit doesn't work on the Package.
+    protected int $requestLimit = 1;
+
+    // Number of times to be run per minute
+    protected int $requestRate = 10;
+
+    protected int $page = 1;
+    protected int $pageCount;
+
+    /**
+     * @var UnionCloud
+     */
+    private $repository;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
```

Comment:
Laravel will run this on every single request as it registers the commands, so we should limit what's being done here. The main issue will be injecting UnoinCloud.

By typehinting the interface `BristolSU\UnionCloud\UnionCloud\UnionCloudContract` (not the concretion) in the handle method, the UnionCloud class will only be created when being used